packages/pn-personafisica-webapp/src/redux/notification/reducers.ts

```diff
@@ -80,12 +84,42 @@ const notificationSlice = createSlice({
     clearDowntimeLegalFactData: (state) => {
       state.downtimeLegalFactUrl = '';
     },
-    setF24Payments: (state, action) => {
-      state.paymentInfo = action.payload;
-    },
   },
   extraReducers: (builder) => {
     builder.addCase(getReceivedNotification.fulfilled, (state, action) => {
+      const paymentsOfRecipient = action.payload.recipients.find(
+        (recipient) => recipient.taxId === action.payload.currentRecipient.taxId
+      )?.payments;
+
+      if (paymentsOfRecipient) {
```

Comment:
maybe here you can create two functions in notification.utility.tsx. One that get the pagoPaF24 payments and one that get only the f24

Reply:
Ok, but I can't use them here because I've to cast the pagoPa property